build/scripts/DownloadVCLibsDesktop.ps1

```diff
@@ -0,0 +1,27 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform = "x64",
+    [string]$OutputDirectory = "D:\WindowsAppSDK\build\scripts"
+)
+
+$ProgressPreference = "SilentlyContinue"
+$ErrorActionPreference = "Stop"
+
+$outputPath = Join-Path $OutputDirectory "Microsoft.VCLibs.$Platform.14.00.Desktop.appx"
+if(!(Test-Path $OutputDirectory))
```

Comment:
Should this be

```
if (!(TestPath foo))
```

or

```
if (-not(TestPath foo))
```

Which is recommended practice? I ask as I've usually seen `-not` but I don't know which is best practice

Reply:
I'm pretty -not is the recommended practice. Fixing.